community/abiword/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=abiword
 pkgver=3.0.2
-pkgrel=4
+pkgrel=5
```

Comment:
Is a `pkgrel` bump needed in this case? Is there an issue with the existing abiword-plugin-command package?

Reply:
Thanks Ikke. Removed pkgrel bump